packages/compile-vyper/parser.js

```diff
@@ -0,0 +1,72 @@
+const debug = require("debug")("compile-vyper:parser");
+const OS = require("os");
+
+function parseImports(body) {
+  // WARNING: We're going to do this with crudely with regexes!!
```

Comment:
Not that it matters too much but your comment is grammatically incorrect

Reply:
Even the comments are done with crudely, I guess XD

(But yeah I can go back and fix this)